supplier/migrations/0001_initial.py

```diff
@@ -0,0 +1,41 @@
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Supplier',
+            fields=[
+                ('user_name', models.CharField(help_text='User Name',
+                                               max_length=32,
+                                               primary_key=True,
+                                               serialize=False,
+                                               validators=[django.core.validators.MinLengthValidator(1)])),
+                ('first_name', models.CharField(help_text='First Name',
+                                                max_length=32,
+                                                null=True)),
+                ('last_name', models.CharField(help_text='Last Name',
+                                               max_length=32,
+                                               null=True)),
+                ('password', models.CharField(help_text='Password',
+                                              max_length=16,
+                                              validators=[django.core.validators.MinLengthValidator(1)])),
+                ('user_type', models.CharField(help_text='User Type - Supplier/Client',
```

Comment:
1. This should probably be an ENUM, not a char. 
2. why is it needed?

Reply:
1. ill figure it out how to do so
2. the user type field is inherited from user class, it might be client aswell.

Reply:
It cannot be a client, it is a Supplier.
Supplier is a supplier User, Client is a client User, User is User; It doesn't need its own identity.
Think about regular OOP, would a mammal have a "type" field that includes dog, cat, etc... ? probably not. you'll know which mammal it is by the class that inherits from it.

Reply:
> Think about regular OOP, would a mammal have a "type" field that includes dog, cat, etc... ? probably not. you'll know which mammal it is by the class that inherits from it.

user type was added with the thought identifying the class easier, lets say we have a container of Users, how will we know if its supplier or client? of coures theres the build it method in python, but it seems that all the team agreed upon adding the field user type to identify, its even in the ERD.
if its better to identify with the built in method then we need to delete it from the ERD aswell.